src/frontend/src/lib/components/tokens/TokenGroupCard.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+	import { slide } from 'svelte/transition';
+	import TokenCard from '$lib/components/tokens/TokenCard.svelte';
+	import TokenCardContent from '$lib/components/tokens/TokenCardContent.svelte';
+	import TokenCardWithOnClick from '$lib/components/tokens/TokenCardWithOnClick.svelte';
+	import TokenCardWithUrl from '$lib/components/tokens/TokenCardWithUrl.svelte';
+	import { TOKEN_GROUP } from '$lib/constants/test-ids.constants';
+	import { SLIDE_PARAMS } from '$lib/constants/transition.constants';
+	import type { TokenUiGroup } from '$lib/types/token';
+	import type { CardData } from '$lib/types/token-card';
+
+	export let tokenGroup: TokenUiGroup;
+
+	let isOpened = false;
```

Comment:
`isExpanded` for consistency? I noticed that term used in another PR.

Reply:
yes, makes sense, it is kind of a `dropdown`